### ISSUE TITLE
Cleaning NGINX Management Suite repository after installation prevents nms-nap-compiler manual install

### ISSUE THREAD

**Issue by fabriziofiorucci** [nginxinc/ansible-role-nginx-management-suite#59]
### Describe the bug

After installing NGINX Management Suite on Debian/Ubuntu the file `/etc/apt/sources.list.d/nms.list` is removed through `tasks/nms/remove-certs.yaml` together with the NGINX license certificate and key. This makes it impossible to find & install `nms-nap-compiler-*`.

### To reproduce

1. Deploy the Ansible NGINX Management Suite role using `playbook.yml`
2. Run `apt-cache search nms-nap-compiler`
3. No packages are found

### Expected behavior

After installing NGINX Management Suite using this role it should be possible to `apt-get install` the `nms-nap-compiler` module without having to recreate `/etc/apt/sources.list.d/nms.list`

### Your environment

- Ansible NGINX Management Suite role 0.3.0
- Version of Ansible: `ansible [core 2.12.10]`
- Target deployment platform: Ubuntu 20.04

### Additional context

One potential improvement might be having the ability to specifiy one or more `nms-nap-compiler` versions to be installed, directly in the playbook.

**Comment by jswongF5** [comment]
You are right that I delete the NGINX Certs but I don't think I delete the license though.. I remove the certs by default mimicking the **ansible-role-nginx** role. I didn't make much of an issue out of this behavior because it made sense to be on the safer side so the certs can't be stolen if a host were somehow compromised.

You can bypass this behavior though.. When you run your role, you can make sure to set the following variable `nms_remove_certs` to `false`
```yaml
- name: Install NMS Playbook
  tasks:
    - name: Install NMS
      ansible.builtin.include_role:
        name: nginxinc.nginx_management_suite
      vars:
        nms_remove_certs: false
```

You are correct that including the ability to install `nms-nap-compiler` is an improvement. At the moment, this role does not do this.
